client/app/scripts/actions/app-actions.js

```diff
@@ -635,13 +635,15 @@ function updateFromNodesDeltaBuffer(dispatch, state) {
 }
 
 export function clickResumeUpdate() {
-  return (dispatch, getServiceState) => {
+  return (dispatch, getState) => {
     dispatch({
       type: ActionTypes.CLICK_RESUME_UPDATE
     });
+    // TODO: Find a better way to do this (see the comment above).
+    const state = getState().scope || getState();
```

Comment:
@fbarl It looks like the `clickResumeUpdate` action is called by the `PauseButton` as well. Should that not also be wrapped in a `<CloudFeature />`? That would solve this issue.